plugins/pageview/tx_dlf_pageview.js

```diff
@@ -54,6 +54,13 @@ var dlfViewer = function(settings){
      */
     this.fulltexts = dlfUtils.exists(settings.fulltexts) ? settings.fulltexts : [];
 
+    /**
+     * Fulltext mimetype (e.g.  application/tei+xml or text/xml)
```

Comment:
`this.teifulltext` is a boolean, which is `true` for TEI fulltexts and `false` otherwise. It doesn't hold the actual MIMETYPE...